src/web/mjs/connectors/Mangafest.mjs

```diff
@@ -0,0 +1,94 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Mangafest extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'mangafest';
+        super.label = 'Mangafest X';
+        this.tags = [ 'indonesian', 'webtoon'];
+        this.url = 'https://mangafest.net';
+    }
+
+    async _getMangas() {
+        let request = new Request(new URL('/category/manga/', this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'nav.navigation.pagination div.nav-links a.page-numbers:not(.next)');
+        const pages = parseInt(data[data.length-1].pathname.match(/\d+\/$/));
+
+        let mangas = [];
+        for (let page = 1; page <= pages; page++) {
+            request = new Request(new URL('/category/manga/page/'+page+'/', this.url), this.requestOptions);
+            data = await this.fetchDOM(request, 'div.post_image a');
+            mangas.push(...data.map(manga => {
+                return {
+                    id: this.getRootRelativeOrAbsoluteLink(manga.pathname, request.url),
+                    title: manga.getAttribute('aria-label').trim()
+                };
+            }));
+        }
+
+        return mangas;
+    }
+
+    async _getChapters(manga) {
+        // website provides mangas as whole
+        return [
+            {
+                id: manga.id,
+                title: manga.title
+            }
+        ];
+    }
+
+    async _getPages(manga) {
+        let request = new Request(new URL(manga.id, this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'div#ngy2p');
+        const flickr_data = JSON.parse(data[0].getAttribute('data-nanogallery2-portable'));
+
+        let url = new URL('https://api.flickr.com/services/rest/?&method=flickr.photosets.getPhotos&api_key=2f0e634b471fdb47446abcb9c5afebdc&format=json');
```

Comment:
For readability the parameters `method`, `api_key` and `format` might be set with `url.searchParams.set()` as well instead of appending them to the USVString